### PR TITLE
Nemo/Added moment.js

### PR DESCRIPTION
**Pull request by nemodjuric**
Connects to #964

### CODE REVIEW

File: client/npm-shrinkwrap.json [hunk 1]
```diff
@@ -199,811 +199,811 @@
...
     "babel-helper-hoist-variables": {
       "version": "6.24.1",
-      "from": "babel-helper-hoist-variables@>=6.24.1 <7.0.0",
+      "from": "https://registry.npmjs.org/babel-helper-hoist-variables/-/babel-helper-hoist-variables-6.24.1.tgz",
       "resolved": "https://registry.npmjs.org/babel-helper-hoist-variables/-/babel-helper-hoist-variables-6.24.1.tgz",
       "dependencies": {
         "babel-runtime": {
           "version": "6.23.0",
-          "from": "babel-runtime@^6.22.0",
+          "from": "https://registry.npmjs.org/babel-runtime/-/babel-runtime-6.23.0.tgz",
```
Comment (amprokop) [inline client/npm-shrinkwrap.json]: i don't remember. did we determine that this does or does not bypass the npm cache server? cc @joofsh @askldjd

**Comment by amprokop** [comment]
Had a question about this.

**Comment by NickHeiner** [comment]
Wouldn't we typically add deps in the PR where they are actually being used?

File: client/npm-shrinkwrap.json (hunk 1 above)
Comment (askldjd) [inline client/npm-shrinkwrap.json]: There is a `sed` call that replace the npm registry URL to our local server. I think the replace should work, but need to double-check

**Comment by askldjd** [comment]
That's true, adding dep without seeing usage is a bit strange. However, moment.js is so common, I have no problem with this addition

**Comment by amprokop** [comment]
True enough for the dependency add. I think @nemodjuric might have wanted to get it in before finishing the PR that uses moment because another dev also wants to use moment.

**Pull request merged**